src/types.ts

```diff
@@ -12,6 +14,7 @@ export enum TagAttributes {
   LOCALE = 'data-contentful-locale',
 }
 
+// TODO: can we add sys and optional typename to the Entity?
 export type Entity = Record<string, unknown>;
```

Comment:
Should we have 2 types? `GraphqlEntity` & `Entity` in this case

Reply:
GraphQL entity would be the one with __typename, but also the REST one would need the `sys` information, so I'm still not sure whats the best way to define/solve this 💭

Reply:
Maybe we gain more insights with the REST implementation 👍